components/pages/home/Navbar.js

```diff
@@ -0,0 +1,191 @@
+import Image from "next/image";
+import { Icon } from "@iconify/react";
+import Link from "next/link";
+import { useState } from "react";
+
+const Navbar = () => {
+  const [isDarkMode, setisDarkMode] = useState(false);
```

Comment:
Penamaan variable buat state sama setternya harus camelCase

```suggestion
  const [isDarkMode, setIsDarkMode] = useState(false);
```